src/components/Auth/Lazy.tsx

```diff
@@ -0,0 +1,20 @@
+import { h } from 'preact'
+import { localised } from '../../locales'
+import { asyncComponent } from '~utils/components'
+import style from './style.scss'
+
+const Loading = localised(({ translate }) => (
+  <div className={style.loading}>
+    {translate('generic.lazy_load_placeholder')}
+  </div>
+))
+
+const AsyncCrossDevice = asyncComponent(
+  //@ts-ignore
+  () => import(/* webpackChunkName: "vendor" */ './Auth.js'),
```

Comment:
I would rename this to something more specific
```suggestion
  () => import(/* webpackChunkName: "authVendor" */ './Auth.js'),
```